bip-musig2/Makefile

```
@@ -0,0 +1,5 @@
+.PHONY: check
+check: key_agg_vectors.json nonce_agg_vectors.json nonce_gen_vectors.json sig_agg_vectors.json sign_verify_vectors.json tweak_vectors.json reference.py gen_vectors_helper.py
+	mypy reference.py > /dev/null
+	python reference.py
+	python gen_vectors_helper.py > /dev/null
```

This fails on my machine unless you specify `python3`

```suggestion
	python3 reference.py
	python3 gen_vectors_helper.py > /dev/null
```

---

Python 2 is EOL since Jan 2020, but ok ;)

---

My bad, I wasn't aware of the EOL. This seems like a configuration problem on my end. Feel free to change it back if you think it maeks sense!